pkg/controller/ibmblockcsi/syncer/csi_controller.go

```diff
@@ -50,6 +50,7 @@ const (
 )
 
 var controllerContainerHealthPort = intstr.FromInt(controllerContainerHealthPortNumber)
+var TopologyEnabled bool
```

Comment:
Same

Reply:
it already initiated in that stage

Reply:
What do you mean "at the stage"? What happens in Go if we access an uninitialized variable? Is there a default value?

Reply:
its initiated in [main](https://github.com/IBM/ibm-block-csi-operator/blob/c9f1d2b9256a44368432c7beafe5c30fabb8aa79/cmd/manager/main.go#L140)

Reply:
As I [said](https://github.com/IBM/ibm-block-csi-operator/pull/192#discussion_r667518312) previously, We don't need to remember these things.

Reply:
done